sway/util.c

```diff
@@ -0,0 +1,80 @@
+#define _POSIX_C_SOURCE 200809L
+#include <signal.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <unistd.h>
+#include <wayland-server.h>
+
+#include "log.h"
+#include "util.h"
+
+struct wl_client * spawn_wl_client(char * const cmd[], struct wl_display * display, struct wl_listener *client_destroy)
+{
```

Comment:
Style: `{` on the same line as the function prototype

Reply:
Fixed